pyaedt/edb_core/edb_data/sources.py

```diff
@@ -437,3 +439,51 @@ def rvalue(self, value):
     def source_type(self):
         """Source type."""
         return self._source_type
+
+
+class Port:
+    """Manages port properties"""
```

Comment:
Documentation for the initializer is missing.
If possible, could you also add an example in the docstring.

Reply:
Can you help me with a template?